packages/@aws-cdk/aws-ec2-alpha/awslint.json

```diff
@@ -2,6 +2,7 @@
     "exclude": [
       "from-method:@aws-cdk/aws-ec2-alpha.VpcV2",
       "attribute-tag:@aws-cdk/aws-ec2-alpha.RouteTable.routeTableId",
-      "from-method:@aws-cdk/aws-ec2-alpha.SubnetV2"
+      "from-method:@aws-cdk/aws-ec2-alpha.SubnetV2",
+      "from-method:@aws-cdk/aws-ec2-alpha.Route"
```

Comment:
For my knowledge, what does these two lines do?

Reply:
this is enforced from `awslint` to have the import methods in classes extending `Resource` , since we haven't introduced it yet these are part of exceptions